pymc/sampling.py

```diff
@@ -2093,6 +2094,73 @@ def sample_prior_predictive(
     return pm.to_inference_data(prior=prior, **ikwargs)
 
 
+def draw(
+    vars: Union[Variable, Sequence[Variable]],
+    draws=1,
+    mode: Optional[Union[str, Mode]] = None,
+    **kwargs,
+) -> List[np.ndarray]:
+    """Draw samples for one variable or a list of variables
+
+    Parameters
+    ----------
+    vars : Union[Variable, Sequence[Variable]]
+        A variable or a list of variables for which to draw samples.
+    draws : int
+        Number of samples needed to draw. Detaults to 500.
+    mode : Optional[Union[str, Mode]]
+        The mode used by ``aesara.function`` to compile the graph.
+    kwargs:
+        Keyword arguments for :func:`pymc.aesara.compile_pymc`
+
+    Returns
+    -------
+    List[np.ndarray]
+        A list of numpy arrays.
+
+    Examples
+    --------
+    .. code-block:: python
```

Comment:
You need a blank line between the directive definition and the content

Reply:
Hi, can you be more specific? Thanks

Reply:
Just updated the code-block, hope that it should be alright now :)